experiments/fiction_experiments.hpp

```diff
@@ -49,15 +49,61 @@ inline constexpr uint64_t xor5_maj         = 0b000000000000000000000000000000000
 inline constexpr uint64_t parity           = 0b0000000000000000000000000000000000000000010000000000000000000000;
 inline constexpr uint64_t fontes18         = 0b0000000000000000000000000000000000000000011111111111111110000000;
 
-inline constexpr uint64_t all              = 0b0000000000000000000000000000000000000000011111111111111111111111;
+
```

Comment:
We can delete my version of "fiction_experiments.hpp". It was just an experiment a few weeks ago. I was unsure if it is good to delete a header-file before conducting the PR.